assets/lokomotive-kubernetes/bootkube/resources/charts/webhook-admission/values.yaml

```diff
@@ -0,0 +1,33 @@
+certificates:
+  caKey:
+  caCert:
+
+namespace: kube-system
+
+deployment:
+  replicaCount: 1
+  image:
+    repository: kinvolk/mutating-admission-controller
```

Comment:
Can you please point to the source code of this controller? It needs to be reviewed as well.

Reply:
I will create a PR for this in operator repository